src/templates/finops-hub/modules/dataFactory.bicep

```diff
@@ -0,0 +1,715 @@
+        name: 'Execute'
+        type: 'ExecutePipeline'
+        dependsOn: []
+        userProperties: []
+        typeProperties: {
+          pipeline: {
+            referenceName: '${safeExportContainerName}_transform'
+            type: 'PipelineReference'
+          }
+          waitOnCompletion: false
+          parameters: {
+            folderName: {
+              value: '@pipeline().parameters.folderName'
+              type: 'Expression'
+            }
+            fileName: {
+              value: '@pipeline().parameters.fileName'
+              type: 'Expression'
+            }
+          }
+        }
+      }
+    ]
+    parameters: {
+      folderName: {
+        type: 'string'
+      }
+      fileName: {
+        type: 'string'
+      }
+    }
+    annotations: []
+  }
+}
+
+//------------------------------------------------------------------------------
+// Export container transform pipeline
+// Trigger: pipeline_extractExport
+//
+// Converts CSV files to Parquet or .CSV.GZ files.
+//------------------------------------------------------------------------------
+
+resource pipeline_transformExport 'Microsoft.DataFactory/factories/pipelines@2018-06-01' = {
+  name: '${safeExportContainerName}_transform'
+  parent: dataFactory
+  dependsOn: [
+    dataset_msexports
+    dataset_ingestion
+  ]
+  properties: {
+    activities: [
+      // (start) -> Wait -> Scope -> Metric -> Date -> File -> Folder -> Delete Target -> Convert CSV -> Delete CSV -> (end)
+      // Wait
+      {
+        name: 'Wait'
+        type: 'Wait'
+        dependsOn: []
+        userProperties: []
+        typeProperties: {
+          waitTimeInSeconds: 60
+        }
+      }
+      // Set Scope
+      {
+        name: 'Set Scope'
+        type: 'SetVariable'
+        dependsOn: [
+          {
+            activity: 'Wait'
+            dependencyConditions: [
+              'Completed'
+            ]
+          }
+        ]
+        userProperties: []
+        typeProperties: {
+          variableName: 'scope'
+          value: {
+            value: '@replace(split(pipeline().parameters.folderName,split(pipeline().parameters.folderName, \'/\')[sub(length(split(pipeline().parameters.folderName, \'/\')), 4)])[0],\'${exportContainerName}\',\'${ingestionContainerName}\')'
+            type: 'Expression'
+          }
+        }
+      }
+      // Set Metric
+      {
+        name: 'Set Metric'
+        type: 'SetVariable'
+        dependsOn: [
+          {
+            activity: 'Set Scope'
+            dependencyConditions: [
+              'Completed'
+            ]
+          }
+        ]
+        userProperties: []
+        typeProperties: {
+          variableName: 'metric'
+          value: {
+            // TODO: Parse metric out of the export path with self-managed exports -- value: '@first(split(split(pipeline().parameters.folderName, \'/\')[sub(length(split(pipeline().parameters.folderName, \'/\')), 4)], \'-\'))'
+            value: 'amortizedcost'
```

Comment:
FYI - Hard-coding for v0.0.1. Left the original code commented out so we can bring it back in v0.0.2 with self-managed exports.

_Side note: I just saw the PR to allow dashes and underscores in export names, so that should be fixed in the next week or two._

Reply:
Love it.  Tested it.  Looks great.